subsys/bluetooth/mesh/model_utils.h

```diff
@@ -16,6 +16,13 @@
 #include <bluetooth/mesh/model_types.h>
 #include <bluetooth/mesh/gen_dtt_srv.h>
 
+#if IS_ENABLED(CONFIG_EMDS)
+/**
+ * @brief Create model key to identify the model, where @p mod is the model.
+ */
+#define MODEL_KEY(mod) (((uint16_t)mod->elem_idx << 8) | mod->mod_idx)
```

Comment:
Could you rename `MODEL_KEY` to something else? model unique emds id or similar. If I hadn't knew design idea I would decide it is related to encryption\decryption on model layer in mesh.